src/editor/EditorCommandHandlers.js

```diff
@@ -1137,6 +1143,38 @@ define(function (require, exports, module) {
         return result.promise();
     }
 
+  function _execCommand(cmd) {
+		window.document.execCommand(cmd);
+	}
+	
+	var _execCommandCut = (function() {
+		if(document.queryCommandSupported("cut")) {
+			return function() {
+				_execCommand("cut");
+			};
+		} else {
+			Dialogs.showModalDialogUsingTemplate(dialogTemplate);
+			return ignoreCommand;
+		}
```

Comment:
This pattern is wrong, here, and also in the code below.  You need to *always* return a function from the anonymous function being run on line 1150.  Basically, you are making `_execCommandCut` a function, but you won't know which one of the two possible options until run time when you check on the value of `document.queryCommandSupported("cut")`.

It should be something like this:

```js
var _execCommandCut = (function() {
    if(document.queryCommandSupported("cut")) {
        return function() {
            _execCommand("cut");
        };
    }

    return function() {
        Dialogs.showModalDialogUsingTemplate(dialogTemplate);
        return ignoreCommand();
    };
}());
```